lib/RateLimiterSQLite.js

```diff
@@ -0,0 +1,214 @@
+const RateLimiterStoreAbstract = require("./RateLimiterStoreAbstract");
+const RateLimiterRes = require("./RateLimiterRes");
+
+class RateLimiterSQLite extends RateLimiterStoreAbstract {
+  /**
+   * @callback callback
+   * @param {Object} err
+   *
+   * @param {Object} opts
+   * @param {callback} cb
+   * Defaults {
+   *   ... see other in RateLimiterStoreAbstract
+   *
+   *   storeClient: sqliteClient, // SQLite database instance
+   *   tableName: 'string',
+   * }
+   */
+  constructor(opts, cb = null) {
+    super(opts);
+
+    this.client = opts.storeClient;
+    this.tableName = opts.tableName;
+    this.tableCreated = opts.tableCreated || false;
+    this.clearExpiredByTimeout = opts.clearExpiredByTimeout;
+
+    // Create table on constructor
+    if (!/^[A-Za-z0-9_]*$/.test(this.tableName)) {
+      throw new Error("Table name must contain only letters and numbers");
```

Comment:
@muco-rolle Should we call `cb(new Error(...))` if `cb` is function? This will help to implement better error handling.